controllers/configurationpolicy_controller_suite_test.go

```diff
@@ -43,7 +43,7 @@ func TestMain(m *testing.M) {
 		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "deploy", "crds")},
 	}
 
-	err := policiesv1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
These changes should have been part of a different commit but I'll let it slide this time. :slightly_smiling_face:

Reply:
I was being lazy last night. I'm going to spend a bit more time figuring out commits this morning.

Reply:
Don't spend too much time on this. Just try to separate your commits in future PRs.